packages/orchestrator-ui-components/src/components/WfoTable/utils/columns.ts

```diff
@@ -66,3 +66,5 @@ export const getSortDirectionFromString = (
             return undefined;
     }
 };
+
+export const WFO_TABLE_COLOR_FIELD = 'color';
```

Comment:
Make this more descriptive. I think it means statusColorField right?

Reply:
Renamed to WFO_STATUS_COLOR_FIELD